gulp/sass.js

```diff
@@ -96,7 +96,7 @@ module.exports = (gulp, plugins, blueprint) => {
             // package the variables list for consumption -- no imports or functions
             .pipe(plugins.stripCssComments())
             .pipe(plugins.replace(/\n{3,}/g, "\n\n"))
-            .pipe(plugins.replace(/(@import|\/\/).*\n+/g, ""))
+            .pipe(plugins.replace(/(@import|\/\/).*\r\n+/g, ""))
```

Comment:
What if we use `\r?\n` to indicate that the `\r` is optional but the `\n` is required?